src/crypto/DeviceList.js

```diff
@@ -165,6 +279,22 @@ export default class DeviceList {
         return res;
     }
 
+    /**
+     * Get the stored device data for a user, in raw object form
+     *
+     * @param {string} userId the user to get data for
+     *
+     * @return {Object} userId->deviceId->{object} devices, or null if
+     * there is no data for this user.
+     */
+    getRawStoredDevicesForUser(userId) {
```

Comment:
This got added because `crypto/index.js` was rifling around in the session store itself to get data out, so rather than make it rifle around in indexeddb instead (which wouldn't necessarily be up to date anyway), I've given it a function to get the data it wants from the DeviceList class.

Reply:
Why not just use `getStoredDevice` ?

Reply:
It could do, but `getStoredDevice` returns a `DeviceInfo` so we'd have to adapt the code to manipulate the `DeviceInfo` class and save it back rather than just manipulating the data as stored.